staging/src/k8s.io/client-go/rest/client.go

```diff
@@ -100,11 +100,14 @@ type RESTClient struct {
 
 	// Set specific behavior of the client.  If not set http.DefaultClient will be used.
 	Client *http.Client
+
+	// RequestTimeout specifies a time limit for requests made by the HTTP Client
+	RequestTimeout time.Duration
```

Comment:
does this have to be public?

Reply:
no, I don't think so. Will change to private.